modules/ingest-common/src/test/java/org/elasticsearch/ingest/common/SortProcessorTests.java

```diff
@@ -276,7 +276,7 @@ public void testSortNullValue() throws Exception {
     }
 
     public void testDescendingSortWithTargetField() throws Exception {
-        IngestDocument ingestDocument = RandomDocumentPicks.randomIngestDocument(random());
+        IngestDocument ingestDocument = RandomDocumentPicks.randomIngestDocument(random(), new HashMap<>());
```

Comment:
ah yes. I didn't realize this source object was copied in the constructor. I remember this being an issue before so I avoided it. will update all